src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MovePersister.kt

```diff
@@ -90,14 +92,26 @@ class MovePersister(
 
           eventsToSave += moveEvents + journeyEvents
 
-          if (counter++ % 1000 == 0) {
-            logger.info("Persisted $counter moves out of ${moves.size} (flushing moves to the database).")
-            save()
+          Result.runCatching { save() }
+            .onSuccess { counter++ }
+            .onFailure {
```

Comment:
Undecided on how to report the failed move at the moment so just logging for now.